src/main/java/com/burgstaller/okhttp/CacheKeyProvider.java

```diff
@@ -1,16 +1,18 @@
 package com.burgstaller.okhttp;
 
-import okhttp3.Request;
+public interface CacheKeyProvider<T> {
```

Comment:
can you please keep the interface documentation?

Reply:
done.